ArduPlane/reverse_thrust.cpp

```diff
@@ -137,3 +137,25 @@ int16_t Plane::get_throttle_input(bool no_deadzone) const
     }
     return ret;
 }
+
+/*
+  return control in from the radio throttle channel with curve giving mid-stick equal to TRIM_THROTTLE.
+ */
+int16_t Plane::get_centered_throttle_input(bool no_deadzone) const
+{
+    int16_t ret;
```

Comment:
```suggestion
    if ((g2.flight_options & FlightOptions::CENTER_THROTTLE_TRIM) == 0) {
        return get_throttle_input(no_deadzone)
    }
    int16_t ret;
```

Reply:
In fact you might even be able to do this and skip the duplicate fetching of in with/without deadzone. I'm not sure how the reversing would work out tho.
```suggestion
    int16_t ret = get_throttle_input(no_deadzone);
    if ((g2.flight_options & FlightOptions::CENTER_THROTTLE_TRIM) == 0) {
        return ret;
    }
    
    // optionally adjust throttle trim point(midstick) to output TRIM_THROTTLE ,non-reverse throttles only
```